nodes/vector/turbulence.py

```diff
@@ -124,21 +123,12 @@ def process(self):
         m_freq = inputs['Frequency'].sv_get()[0]
         m_seed = inputs['Random seed'].sv_get()[0]
 
-        #if verts and verts[0]:
-
         maxlen = len(verts[0])
```

Comment:
this should test the length of incoming verts... so if there are three circle vert_lists coming in `len(verts) == 3`